assets/script.js

```diff
@@ -0,0 +1,144 @@
+const workingHours = [
+  {
+    label: "9AM",
+    key: 9,
+  },
+  {
+    label: "10AM",
+    key: 10,
+  },
+  {
+    label: "11AM",
+    key: 11,
+  },
+  {
+    label: "12AM",
+    key: 12,
+  },
+  {
+    label: "1PM",
+    key: 13,
+  },
+  {
+    label: "2PM",
+    key: 14,
+  },
+  {
+    label: "3PM",
+    key: 15,
+  },
+  {
+    label: "4PM",
+    key: 16,
+  },
+  {
+    label: "5PM",
+    key: 17,
+  },
+];
+
+const timeBlocks = $("#time-blocks");
+
+//accessing date and time
+//const currentDateStamp = moment();
+
+const renderDate = () => {
+  return moment().format("dddd, MMMM Do YYYY HH:mm");
+};
+
+$("#currentDay").text(renderDate);
+
+$("#saveBtn").click(function () {});
```

Comment:
I presume this is being handled elsewhere now - make sure to remove any redundant code when you've finished.